### PR TITLE
restartcloud changes from cloudfoundry team

### PR DESCRIPTION
**Pull request by bmwiedemann**
2nd commit change is only partially tested

### CODE REVIEW

File: scripts/lib/mkcloud-onhost.sh [hunk 1]
```diff
@@ -54,10 +54,9 @@ done
 iptables_unique_rule PREROUTING -t nat -p tcp --dport \$(( $cloud_port_offset + 6080 )) \\
     -j DNAT --to-destination $net_public.2
 
-# need to delete+insert on top to make sure our ACCEPT comes before libvirt's REJECT
-for x in D I ; do
-    iptables -\$x FORWARD -d $net_admin.0/24 -j ACCEPT
-    iptables -\$x FORWARD -d $net_public.0/24 -j ACCEPT
+# delete libvirt's REJECT rules to allow public net
```
Comment (toabctl) [inline scripts/lib/mkcloud-onhost.sh]: can this be done somehow in libvirt via the <filter> element?

Comment (bmwiedemann) [inline scripts/lib/mkcloud-onhost.sh]: I followed https://www.ibm.com/support/knowledgecenter/en/linuxonibm/liaat/liaatsecnetfilt.htm and tried adding below <interface type='network'>
```
<filterref filter='allow-ipv4'/>
```
and tried
virsh nwfilter-edit clean-traffic
to clean the reject rules

but that did not get rid of the default reject rules. Any further hints highly welcome.

Comment (toabctl) [inline scripts/lib/mkcloud-onhost.sh]: Hm. I also do not know if that would work. so aproving this PR...

**Comment by bmwiedemann** [comment]
Should do some extra manual testing, to see if/when libvirt adds back those REJECT rules.

**Comment by bmwiedemann** [comment]
closing

**Pull request closed without merging**